pytrip/cube.py

```diff
@@ -462,16 +463,23 @@ def create_dicom_base(self):
         ds.SOPInstanceUID = '1.2.3'  # !!!!!!!!!!
         ds.StudyInstanceUID = '1.2.3'  # !!!!!!!!!!
         ds.FrameofReferenceUID = '1.2.3'  # !!!!!!!!!
-        ds.StudyDate = '19010101'  # !!!!!!!
-        ds.StudyTime = '000000'  # !!!!!!!!!!
+        ds.SeriesInstanceUID = UID.generate_uid(prefix="2.25.")
+        ds.StudyDate = datetime.datetime.today().strftime('%Y%m%d')
+        ds.StudyTime = datetime.datetime.today().strftime('%H%M%S')
         ds.PhotometricInterpretation = 'MONOCHROME2'
         ds.SamplesPerPixel = 1
         ds.ImageOrientationPatient = ['1', '0', '0', '0', '1', '0']
         ds.Rows = self.dimx
         ds.Columns = self.dimy
         ds.SliceThickness = str(self.slice_distance)
-
         ds.PixelSpacing = [self.pixel_size, self.pixel_size]
+
+        # Add eclipse friendly IDs
```

Comment:
No, that does not work, I tried.
These attributes do not exist by default in the pydicom object. Creating the attributes, does not tell pydicom to save it when saving the dicom file (also the Tag keys are missing).

Therefore using the add_new() method. Then it works. :)